Sally7/ValueConversion/ToS7Conversions.cs

```diff
@@ -153,6 +137,22 @@ private static int ConvertFromString(in string? value, in int length, in Span<by
                 return 2;
             }
 
+#if NETSTANDARD2_1_OR_GREATER
+            var maxByteCount = Encoding.ASCII.GetMaxByteCount(value.Length);
+            Span<byte> span = maxByteCount <= 256
```

Comment:
I'm not sure if this is overkill here...
Is string-handling in PLC that common? I don't know.

Anyway...if the string is short enough we stack-allocate, otherwise just do a regular allocation. One could rent a buffer from the array-pool, but this makes the code a bit more unreadable though.